extensions/Cake.XmlConfigStructureBuilder.yml

```diff
@@ -10,4 +10,5 @@ Categories:
 - configs generator
 - deployment
 - environment
-TargetCakeVersion: 0.36.0
+TargetCakeVersion: 0.33.0
```

Comment:
This change is due to the fact that we are now analyzing the most recent stable release:

![image](https://user-images.githubusercontent.com/112710/101061992-18d96e00-355f-11eb-956b-56837285189f.png)